protocol/message_test.go

```diff
@@ -0,0 +1,214 @@
+package protocol
+
+import "testing"
+
+var (
+	uname = "alice"
+	key   = []byte("key")
+)
+
+func TestVerifyRegistrationResponseWithTB(t *testing.T) {
+	d, pk := NewTestDirectory(t, true)
+
+	savedSTR := d.LatestSTR().Signature
+	res, _ := d.Register(&RegistrationRequest{
+		Username: uname,
+		Key:      key})
+	if err := res.Verify(uname, key, savedSTR, pk); err != Passed {
```

Comment:
I gather this passes because it's a valid proof of absence, but what's missing is the expectation from the client.  To see why, the next test does exactly the same thing (ignoring the error).  That means the client could be getting a valid proof of absence, when it expects a proof of inclusion, and as long as it's valid, it's none the wiser.